src/pages/spawn/spawnHost/SpawnHostModal.tsx

```diff
@@ -217,12 +248,15 @@ export const SpawnHostModal: React.VFC<SpawnHostModalProps> = ({
         </WideButton>
         <WideButton
           data-cy="spawn-host-button"
-          disabled={!validateSpawnHostForm(formState) || loadingSpawnHost} // @ts-expect-error
+          disabled={
+            !validateSpawnHostForm(formState, !!migrateVolumeId) ||
+            loadingSubmit
+          } // @ts-expect-error
           onClick={spawnHost}
           variant={Variant.Primary}
           key="spawn_host_button"
         >
-          {loadingSpawnHost ? "Spawning Host" : "Spawn"}
+          {loadingSubmit ? "Spawning Host" : "Spawn"}
```

Comment:
The designs actually prompt you to hit next which directs you to a confirmation modal. 
<img width="745" alt="image" src="https://user-images.githubusercontent.com/4605522/197054771-a5771ac7-89c3-4954-a3f7-eafd3e3c55a0.png">
<img width="755" alt="image" src="https://user-images.githubusercontent.com/4605522/197054879-57fa6ff2-8b05-4432-b740-fe34d097d1e2.png">